RELEASE_PROCESS.md

```diff
@@ -32,6 +32,13 @@ In order to release, you will need the following accounts/permissions:
 
 - [ ] Announce your intention to drive the release to #cft-announce and #chef-server on slack.
 
+### Getting the build to be released into current
+
+- Update the version file to the version to be released.
```

Comment:
We only need to do this if we want to bump the minor version rather than just the patch.